README.markdown

```diff
@@ -14,7 +14,9 @@ The final hex file is named `ethersex.hex`.
 
 How to add a new hardware pinning
 =================================
-Use the script at `scripts/add-hardware` to add a new pinning.
+* Make `make menuconfig` and configure hardware / select features
+* Save configuration as .config file
+* Use the script at `scripts/add-hardware` to add a new pinning.
```

Comment:
The add-hardware script calls make menuconfig if no .config exists, so it's pretty self-explanatory